template/src/bin/async_main.rs

```diff
@@ -124,5 +124,5 @@ async fn main(spawner: Spawner) {
         Timer::after(Duration::from_secs(1)).await;
     }
 
-    // for inspiration have a look at the examples at https://github.com/esp-rs/esp-hal/tree/esp-hal-v1.0.0-beta.0/examples/src/bin
+    // for inspiration have a look at the examples at https://github.com/esp-rs/esp-hal/tree/esp-hal-v1.0.0-beta.1/examples/src/bin
```

Comment:
should we parse esp-hal's version from the template toml and replace the version here with the real one automatically? 🤔

Reply:
one of the suggestions in #89